lib/Widget/appointment/addAppointment.dart

```diff
@@ -150,6 +151,10 @@ class _AddAppointmentPageState extends State<AddAppointmentPage> {
                                           padding: const EdgeInsets.fromLTRB(
                                               20, 8, 0, 8),
                                           child: TextFormField(
+                                            inputFormatters: [
+                                              FilteringTextInputFormatter.allow(
+                                                  RegExp("[a-zA-Z0-9]"))
+                                            ],
```

Comment:
I think we should limit the length of input as well rather than just get rid of non-numeric and non-alphabet

Reply:
solved in https://github.com/LaCare-KMUTT/AbdoCare-Web/pull/81/commits/54954161887fa5465028126b9281bfe209f1336a